action.yml

```
@@ -228,7 +231,11 @@ runs:
         if ("${{ steps.validation.outputs.use_custom_url }}" -eq "1") {
           Invoke-Installer -LocalPath "${{ inputs.release-asset-name }}" -InstallArgs ("/quiet")
         } else {
-          Invoke-Installer -URL "https://download.swift.org/${{ inputs.branch }}/windows10/swift-${{ inputs.tag }}/swift-${{ inputs.tag }}-windows10.exe" -InstallArgs ("/quiet")
+          if ("${{ input.build_arch }}" -ne "arm64") {
```

I think that we should make this more generic:

```suggestion
          if ("${{ input.build_arch }}" -eq "amd64") {
```

---

Done